packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -8,6 +8,11 @@ const ADVANCED_GROUP = 'advanced';
 const DISPLAY_GROUP = 'stats';
 const ZERO_CONFIG_GROUP = 'zero-config';
 
+let cli = undefined;
+if (packageExists('webpack')) {
+    cli = require('webpack').cli;
+}
```

Comment:
Why here?

Reply:
Can be refactored as -> `const cli = packageExists('webpack') ? require('webpack').cli : undefined`

Else we get `Error: can't find module 'webpack'`